test/unit/transport/expected_get_message.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><rpc xmlns:if="http://www.cisco.com/nxos:1.0:if_manager" xmlns:nfcli="http://www.cisco.com/nxos:1.0:nfcli" xmlns:nxos="http://www.cisco.com/nxos:1.0" xmlns:vlan_mgr_cli="http://www.cisco.com/nxos:1.0:vlan_mgr_cli" xmlns="urn:ietf:params:xml:ns:netconf:base:1.0" message-id="urn:uuid:UUID-HERE"><get><filter type="subtree"><cmd>show version</cmd></filter></get></rpc>
```

Comment:
the xmlns order changed between now, and the last time I messed with this.  I'm not sure the best way to deal with this.

Should the resulting xml be parsed, and the object diff'd ?
I don't want to ignore all the attributes in the root node because that is where the problem was with lxml 3.2.1

Thoughts / ideas ?